go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/Stanley-Wang910/HealthyUI
```

Comment:
could be wrong, but i think you want to make your module namespace like healthyui/[module domain]
because you're going to want to add a bunch more modules i think that have different responsibilities

anyway, just a suggestion

Reply:
Yeah I was looking into that too. I didn't get around to changing it just yet though